x-pack/plugins/log_explorer/public/components/virtual_columns/column.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiDataGridColumn } from '@elastic/eui';
+import { ContentColumnTooltip } from './column_tooltips/content_column_tooltip';
+import { CONTENT_FIELD } from '../../../common/constants';
+
+export const renderColumn =
+  (field: string) =>
+  ({ column, headerRowHeight }: { column: EuiDataGridColumn; headerRowHeight?: number }) => {
```

Comment:
**nit:** Isn't this type the same signature as defined in `CustomGridColumnsConfiguration` earlier in this PR?
Might be worth extracting and re-using this type for better alignment.